Tasks/NuGetInstaller/task.json

```diff
@@ -29,6 +30,14 @@
             "helpMarkDown": "The path to the Visual Studio solution file or NuGet packages.config",
             "required": "true"
         },
+		{
+            "name": "connectedServiceName",
+            "type": "connectedService:Generic",
```

Comment:
We want to do that eventually but need to put some more cycles into the design. The existing task (mis)uses generic endpoints and I'm OK with leaving it for now.